ECMAScript/react/src/components/QuilttButton.tsx

```diff
@@ -0,0 +1,28 @@
+import { FC, DetailedHTMLProps, ButtonHTMLAttributes } from 'react'
+import { useQuilttConnector } from '..'
+
+type QuilttButtonProps = DetailedHTMLProps<
+  ButtonHTMLAttributes<HTMLButtonElement>,
+  HTMLButtonElement
+> & {
+  connectorId: string
+  connectionId?: string // For Reconnect Mode
+}
+
+export const QuilttButton: FC<QuilttButtonProps> = ({
+  connectorId,
+  connectionId,
+  children,
+  ...props
+}) => {
+  useQuilttConnector()
+
+  return (
+    // eslint-disable-next-line react/no-unknown-property
+    <button quiltt-connector={connectorId} quiltt-connection={connectionId} {...props}>
+      {children}
+    </button>
+  )
+}
```

Comment:
@sirwolfgang without the `ready` in `useQuilttConnector()`, is there not a risk of the button being clickable before the script has loaded?

Reply:
No, I don't think so. I haven’t hooked it up to anything to test; but think it should work the same.

The way that you had ready, was true if the script was attached; not if it was loaded and running.

My version, the script is always attached, so they work the same; I just removed that complexity.

The hook always loads before the component is rendered, so it should be fine. If it’s not then I will have to setup an event system that we don’t current have.